extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -199,6 +200,24 @@ describes.realWin('amp-analytics', {
         expect(linkerStub.calledOnce).to.be.true;
       });
     });
+
+    it('Removes linker listeners.', () => {
```

Comment:
this test is not useful. the bug should have been caught in one of the tests below